locale/Makefile.def

```diff
@@ -40,9 +40,14 @@ uniq-po: ## Unify duplicate translations in .po files
 %.edit.po:
 	touch $@
 
-# Pull translaions from transifex
+# Pull in all languages with at least 50%
+.PHONY: tx-new
+tx-new:
+	tx pull --minimum-perc 50 --all
```

Comment:
My reasoning was that I wanted to separate the "Add new languages" step from "Update languages". For example, I don't know if `tx pull -f --minimum-perc 50 --all` only pulls translations that are >= 50%.

Essentially I'm thinking of the scenario where you have:
* Language XX is added because it's >= 50%
* Later language XX drops below 50%, but there is a new string

In that case you could argue XX should be dropped, but my current intention is that you at least pull in the new string. While I didn't test it, I thought that a single command would ignore the string.

That said, I think `tx-new` now also needs some step to create the `%{DOMAIN}.po` files after pulling so this step may be expanded anyway.